File: example/src/main/java/TestWithExpireTime.java
import com.safframework.rxcache.RxCache;
import com.safframework.rxcache.domain.Record;

/**
 * Created by tony on 2018/10/5.
 */
public class TestWithExpireTime {

    public static void main(String[] args) {

        RxCache.config(new RxCache.Builder());

        RxCache rxCache = RxCache.getRxCache();

        User u = new User();
        u.name = "tony";
        u.password = "123456";
        rxCache.save("test",u,2000);

        try {
            Thread.sleep(2500);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }

        Record<User> record = rxCache.get("test", User.class);

        if (record==null) {
            System.out.println("record is null");
        }
    }
}

File: example/src/main/kotlin/extension/TestWithExpireTime.kt
package extension

import com.safframework.rxcache.RxCache
import com.safframework.rxcache.ext.get
import domain.User

/**
 * Created by tony on 2019-06-26.
 */
object TestWithExpireTime {

    @JvmStatic
    fun main(args: Array<String>) {

        RxCache.config(RxCache.Builder())

        val rxCache = RxCache.getRxCache()

        val u = User()
        u.name = "tony"
        u.password = "123456"
        rxCache.save("test", u, 2000)

        try {
            Thread.sleep(2500)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }

        val record = rxCache.get<User>("test")

        if (record == null) {
            println("record is null")
        }
    }
}